Update BSQ price in documentation (#192)
bisq-network/bisq-docs: compensation.adoc

For example, it might be reasonable to request 50 BSQ for fixing some typos in a doc. But requesting 1000 BSQ for that same task, just because it took you a few hours to read through the doc, will probably be rejected.

> **You can find the exchange rate of BSQ in real time, open trades, trade history and other information in https://bisq.network/markets/?currency=bsq_btc[Bisq Markets^].

Reviewer: ```suggestion
**See the current BSQ price, open trades, trade history and other information on the https://bisq.network/markets/?currency=bsq_btc[Bisq Markets^] page.
```
